capstone/client/src/components/Header.js

```diff
@@ -85,7 +86,7 @@ class Header extends Component {
                   <Menu />
                 </IconButton>
                 <Typography id="typography" variant="h6">
-                  Preferences
+                  {this.props.location.pathname  === '/' ? 'Preferences' : 'Store Recommendations'}
```

Comment:
Actually, probably easier if you can take in the header name as a prop and pass it in. This header file should be generic.